Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -407,7 +407,12 @@ class DropDownMenu
     ul.addEventListener ('mouseover', function (event)
     {
       var eventTarget = DropDownMenu.GetTarget (event, "LI");
+
       ul.querySelectorAll ("li").forEach (b => b.classList.remove (DropDownMenu._itemSelectedClassName));
+
+      if (eventTarget == null)
```

Comment:
This is interesting. Doesn't this mean we have to make DropDownMenu.GetTarget return a nullable string, too?